packages/light-apps/src/globalStyle.tsx

```diff
@@ -1,4 +1,4 @@
-// Copyright 2017-2018 @polkadot/app-accounts authors & contributors
+// Copyright 2017-2018 @polkadot/light-apps authors & contributors
```

Comment:
Nothing critical, but we probably want to publish this via Parity (we have an npm repo for `@parity`) - and copyright probably needs to mirror what we have elsewhere in the Parity repos. (But no need to action here, can just be a bulk replace)